src/utils/constants.ts

```diff
@@ -152,6 +158,19 @@ export const INFO = {
       message: `**Event Slug:** ${event.slug}\n**Event Name:** ${event.name}\n**Channel Started:** <#${event.checkin?.channelId}>`,
     };
   },
+  USER_CHECKIN: (teamName?: string | null) => {
```

Comment:
Why is teamName optional? And if it has to be, you need not add null.

Reply:
registrantSchema has teamName Optional. That is why I have added optional here.

Reply:
Okay so all you need to do is write teanName?:string and it automatically understands that.